index.html

```diff
@@ -0,0 +1,692 @@
+     ></path>
+    </svg>
+   </a>
+        <nav class="nav">
+            <ul class="nav-list">
+                <li>
+                    <a class="subscribe" href="#">SUBSCRIBE NOW</a>
+                </li>
+                <li>
+                    <a class="login" href="#">LOG IN</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main class="container-main">
+        <div class="container-flex">
+            <div class="art-address">
+                <div>
+                    <h1>Space Ripples Reveal Big Bang’s</h1>
+                    <h1>Smoking Gun</h1>
+                </div>
+            </div>
+            <div class="big-img">
+                <img class="main-img" src="assets/imgs/main-img.jpg" alt="new york times" />
+            </div>
+            <div class="mainImg-para">
+                <p>
+                    Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...Rick
+                </p>
+                <small>Friedman for The New York Times</small>
+            </div>
```

Comment:
Here the picture has a description, this means that that picture could be a `<figure>` element and in addition to that could be added a `<figcaption>` as description, this will greatly improve the HTML semantic of your code, please look over the [w3schols](https://www.w3schools.com/tags/tag_figcaption.asp) for a complete explanation.